CalibCalorimetry/HcalAlgos/src/HcalTimeSlew.cc

```diff
@@ -27,7 +27,7 @@ double HcalTimeSlew::delay(double fC, ParaSource source, BiasSetting bias, doubl
   }
   else if (source==Data || source==MC){
     if(isHPD) return std::fmin(cap,tspar0[source-1]+tspar1[source-1]*log(fC+tspar2[source-1]));
-    return std::fmin(cap,tspar0_siPM[source-1]+tspar1_siPM[source-1]*log(fC+tspar2_siPM[source-1]));
+    return cap+tspar0_siPM[source-1];
```

Comment:
I did not notice any significant changes in the hbhereco (M2) times in the last available comparisons https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17094/17152/summary.html
Is this time slew used for what goes in reco?